dependencies.gradle

```diff
@@ -44,6 +44,7 @@ def apt = [
 def external = [
         butterKnife    : "com.jakewharton:butterknife:${versions.butterKnife}",
         dagger         : "com.google.dagger:dagger:${versions.dagger}",
+        daggerAndroid  : "com.google.dagger:dagger-android-support:${versions.dagger}",
```

Comment:
is the dagger android compiler required for this to work properly?

Reply:
I can remove this dependency and make a custom module, similar to `Coffee.kt` added on `kotlinlibrary` project.
Just added this for a more "real looking" project